### PR TITLE
Add HEEWING F405/F405V2 autopilot

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-heewingf405.rst [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+RC Input
+========
+
+The RCIN pin, is passed by an inverter to RX2 (UART2 RX), which by default is mapped to a timer input instead of the UART, and can be used for all ArduPilot supported receiver protocols, except CRSF/ELRS and SRXL2 which require a true UART connection. However, FPort, when connected in this manner, can provide RC without telemetry. 
+
+To allow CRSF and embedded telemetry available in Fport, CRSF, and SRXL2 receivers, the RX2 pin can also be configured to be used as true UART2 RX pin for use with bi-directional systems by setting the :ref:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` to “1” so it becomes the SERIAL2 port's RX input pin.
+
+With this option, :ref:`SERIAL6_PROTOCOL<SERIAL2_PROTOCOL>` must be set to "23", and:
+
+- PPM/SBUS/DSM/SRXL is not supported.
```
Comment (andyp1per) [inline common/source/docs/common-heewingf405.rst:116]: It's only PPM/SBUS - DSM and SRXL are non-inverted so should work fine

File: common/source/docs/common-heewingf405.rst [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+:ref:`BATT_AMP_PERVLT<BATT_AMP_PERVLT>` 26.7
+
+Connecting a GPS/Compass module
+===============================
+
+This board does not include a GPS or compass so an :ref:`external GPS/compass <common-positioning-landing-page>` should be connected in order for autonomous modes to function.
+
+Firmware
+========
+This board does not come with ArduPilot firmware pre-installed (it may at some future point). Because it has not boot pin exposed, DFU loading cannot be used to load standard ArduPilot firmware. Instead a `"transitional" firmware <https://firmware.ardupilot.org/Tools/TransitionalFirmware/HeeWingF405/>`__  can be used and loaded like normal ArduPilot firmware (this is rev 4.3.7 Plane firmware) as a custom firmware file in Mission Planner.
```
Comment (andyp1per) [inline common/source/docs/common-heewingf405.rst:158]: Actually this is not true - it *does* come with AP pre-installed, it's just the board ID they have used is for the MatekF405 so upgrades are not possible. The transitional firmware allows upgrades

**Review by andyp1per — CHANGES_REQUESTED** [review]
Couple of changes

File: common/source/docs/common-heewingf405.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-heewingf405.rst:158]: Ill say it comes with an incorrect bootloader which prevents ArduPilot firmware changes.

File: common/source/docs/common-heewingf405.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-heewingf405.rst:116]: will change

**Pull request merged**